content/tracing/visualization/trace.md

```diff
@@ -23,25 +23,28 @@ View an individual trace to see all of its spans and associated metadata. Each t
 
 {{< img src="tracing/visualization/trace/trace.png" alt="Trace" responsive="true" style="width:90%;">}}
 
-Calculate breakdown of execution time and adjust the color scheme by either **service** or **host**.
+Calculate the breakdown of execution time and adjust the color scheme by either **service** or **host**.
 
 {{< img src="tracing/visualization/trace/service_host_display.png" alt="Service host display" responsive="true" style="width:40%;">}}
 
-List sorts resources by service, not chronologically. Here it's easy to see all resources at a glance and sort by count of spans, average duration, and others:
+Want to look closer at the flame graph? Zoom in by scrolling:
 
-{{< img src="tracing/visualization/trace/trace_list.png" alt="Trace list" responsive="true" style="width:90%;">}}
+{{< img src="tracing/visualization/trace/trace_zoom.gif" alt="Trace Error" responsive="true" style="width:90%;">}}
 
-If you are analyzing an error trace, the error has a specific display if you followed the [special meaning tags rules](#traces-special-meaning-tags):
+The List view sorts resources by service, not chronologically. Here it's easy to see all resources at a glance and sort by count of spans, average duration, and others:
 
-{{< img src="tracing/visualization/trace/trace_error.png" alt="Trace Error" responsive="true" style="width:90%;">}}
+{{< img src="tracing/visualization/trace/trace_list.png" alt="Trace list" responsive="true" style="width:90%;">}}
 
-Want to look closer at the flame graph? Zoom in by scrolling:
+### More Information
 
-{{< img src="tracing/visualization/trace/trace_zoom.gif" alt="Trace Error" responsive="true" style="width:90%;">}}
+{{< tabs >}}
+{{% tab "Span Metadata" %}}
 
-### Traces special meaning tags
+Click on a span in the flame graph to show the metadata below the graph. If there's an error, the stack trace is provided:
+
+{{< img src="tracing/visualization/trace/trace_error.png" alt="Trace Error" responsive="true" style="width:90%;">}}
 
-When [submitting your traces][1] you can add attributes in the `meta` parameter. Some attributes have special meanings that lead to a dedicated display or specific behavior in Datadog:
+If you are analyzing an error trace, the error has a specific display if you follow the special meaning tags rules. When submitting your traces you can add attributes to the `meta` parameter. Some attributes have special meanings that lead to a dedicated display or specific behavior in Datadog:
```

Comment:
r/`If you are analyzing an error trace...`/ `If you are analyzing a trace reporting an error,`